rise/ViewControllers/GuideViewController.m

```diff
@@ -89,6 +101,7 @@ - (void)updateLabels {
         int *poseIndex = [[self.workout.stretches objectAtIndex:exerciseNum] intValue];
         YogaPose *pose = [self.poses objectAtIndex:poseIndex];
         self.poseLabel.text = pose.name;
+        [self.poseLabel sizeToFit];
```

Comment:
The layout of the poseLabel should be auto resized in storyboard, this function call is redundant.